### PR TITLE
Add Rust SDK Repository

### PR DESCRIPTION
**Pull request by GabyUnalaq**

### CODE REVIEW

File: otterdog/eclipse-ankaios.jsonnet [hunk 1]
```diff
@@ -91,5 +91,32 @@ orgs.newOrg('eclipse-ankaios') {
         },
       ],
     },
+    orgs.newRepo('ank-sdk-rust') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      description: "Eclipse Ankaios Rust SDK provides a convenient rust interface for interacting with the Ankaios platform.",
```
Comment (krucod3) [inline otterdog/eclipse-ankaios.jsonnet]:
```suggestion
      description: "Eclipse Ankaios Rust SDK - provides a convenient Rust interface for interacting with the Ankaios platform.",
```

File: otterdog/eclipse-ankaios.jsonnet [hunk 2]
```diff
@@ -91,5 +91,32 @@ orgs.newOrg('eclipse-ankaios') {
...
+      gh_pages_source_branch: "gh-pages",
+      gh_pages_source_path: "/",
+      has_discussions: true,
+      homepage: "https://eclipse-ankaios.github.io/ank-sdk-rust/",
+      topics+: [
+        "automotive",
+        "containers",
+        "orchestration",
+        "ankaios",
+        "sdk",
```
Comment (krucod3) [inline otterdog/eclipse-ankaios.jsonnet:110]: Should we add Rust here:
```suggestion
        "sdk",
        "rust",
```

Comment (GabyUnalaq) [reply otterdog/eclipse-ankaios.jsonnet:110]: Fixed

File: otterdog/eclipse-ankaios.jsonnet (hunk 1 above)
Comment (GabyUnalaq) [reply otterdog/eclipse-ankaios.jsonnet]: Fixed

**Review by krucod3 — APPROVED** [review]
LGTM 👍

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
Applied the changes manually:

```
  ~ repository[name="ank-sdk-python"] {
    ~ description = "Eclipse Ankaios Python SDK provides a convenient python interface for interacting with the Ankaios platform." -> "Eclipse Ankaios Python SDK - provides a convenient Python interface for interacting with the Ankaios platform."
    ~ topics      = [
      - ankaios
      + ankaios
      + python
    ~ ]
  ~ }

  + add repository[name="ank-sdk-rust"] {
    + allow_auto_merge                  = false
    + allow_forking                     = true
    + allow_merge_commit                = true
    + allow_rebase_merge                = true
    + allow_squash_merge                = true
    + allow_update_branch               = false
    + archived                          = false
    + code_scanning_default_setup_enabled = false
    + custom_properties                 = {
    + }
    + default_branch                    = "main"
    + delete_branch_on_merge            = false
    + dependabot_alerts_enabled         = true
    + dependabot_security_updates_enabled = false
    + description                       = "Eclipse Ankaios Rust SDK - provides a convenient Rust interface for interacting with the Ankaios platform."
    + gh_pages_build_type               = "legacy"
    + gh_pages_source_branch            = "gh-pages"
    + gh_pages_source_path              = "/"
    + has_discussions                   = true
    + has_issues                        = true
    + has_projects                      = true
    + has_wiki                          = true
    + homepage                          = "https://eclipse-ankaios.github.io/ank-sdk-rust/"
    + is_template                       = false
    + merge_commit_message              = "PR_TITLE"
    + merge_commit_title                = "MERGE_MESSAGE"
    + name                              = "ank-sdk-rust"
    + private                           = false
    + private_vulnerability_reporting_enabled = false
    + secret_scanning                   = "enabled"
    + secret_scanning_push_protection   = "enabled"
    + squash_merge_commit_message       = "COMMIT_MESSAGES"
    + squash_merge_commit_title         = "COMMIT_OR_PR_TITLE"
    + template_repository               = null
    + topics                            = [
      + "automotive"
      + "containers"
      + "orchestration"
      + "ankaios"
      + "sdk"
      + "rust"
    + ],
    + web_commit_signoff_required       = false
  + }

  + add repo_workflow_settings[repository="ank-sdk-rust"] {
    + enabled                           = true
  + }

  + add environment[name="github-pages", repository="ank-sdk-rust"] {
    + branch_policies                   = [
      + "gh-pages"
    + ],
    + deployment_branch_policy          = "selected"
    + name                              = "github-pages"
    + reviewers                         = []
    + wait_timer                        = "0"
  + }

```

**Pull request merged**